src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinSpan.cs

```diff
@@ -284,15 +285,11 @@ public void Write(Utf8JsonWriter writer)
                 {
                     if (tag.Value is int intValue)
                     {
-                        writer.WriteNumber(tag.Key, intValue);
-                    }
-                    else if (tag.Value is bool boolVal)
-                    {
-                        writer.WriteBoolean(tag.Key, boolVal);
+                        writer.WriteString(tag.Key, intValue.ToString(CultureInfo.InvariantCulture));
```

Comment:
can you confirm if this is what other languages do for zipkin exporter/

Reply:
Java:
- They use a span: https://github.com/open-telemetry/opentelemetry-java/blob/ca85d8f0eb43aa05aa0aa18bdf92fa6b42abbfba/exporters/zipkin/src/main/java/io/opentelemetry/exporters/zipkin/ZipkinSpanExporter.java#L126
- The Span Tags is:
```java
  /**
   * Tags a span with context, usually to support query or aggregation.
   *
   * <p>For example, a tag key could be {@code "http.path"}.
   */
  public Map<String, String> tags() {
    return tags;
  }
```

Python:
https://github.com/open-telemetry/opentelemetry-python/blob/163564a76859f8317a36cfad3da75403610efd9d/exporter/opentelemetry-exporter-zipkin/src/opentelemetry/exporter/zipkin/__init__.py#L200-L213